Jenkinsfile.integration

```diff
@@ -87,6 +87,7 @@ pipeline {
                 anyOf {
                     expression { return DocsChanged }
                     expression { return ToxChanged }
+                    expression { env.BRANCH_NAME == 'master' }
```

Comment:
I admit I do not understand the syntax, but this looks like an OR to me. Should not there be rather "check for master AND doc checks" ?

Reply:
no, this is correct.

So for the master branch, the DocsChange/ToxChanged will always be false, because you cannot compare between master and master and get what has changed :)

This is just for the periodic job which is run off master.

Reply:
Right, so this will build Docs on any push to master or stable/1.0 (regardless of what's changed)?
We should add a comment as to why this is.